orchestrator-swarm/src/main/java/com/sequenceiq/cloudbreak/orchestrator/swarm/DockerClientUtil.java

```diff
@@ -25,10 +25,10 @@
     private static final int MAX_WAIT_FOR_CONTAINER_RETRIES = 60;
```

Comment:
shouldn't this be  a service instead? it could be injected in that case instead of being passed as arguments ...